tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -240,166 +240,138 @@ int main(int argc, char** argv) try
         show_options = show_calibration_data = false;
     }
 
-    log_to_console(RS2_LOG_SEVERITY_FATAL);
-
     for (auto i = 0u; i < device_count; ++i)
     {
-        auto dev = devices[i];
-
-        // Show which options are supported by this device
-        cout << "Device info: \n";
-        if (auto pb_dev = dev.as<playback>())
-            cout << "Playback Device (" << pb_dev.file_name() <<  ")" << std::endl;
-
-        for (auto j = 0; j < RS2_CAMERA_INFO_COUNT; ++j)
+        try
```

Comment:
There are smaller try-catch blocks inside this try block... can we narrow down the scope of this try block to the device creation failure? Which line actually throws?

Reply:
Done. The device creation has been isolated and moved to the start.